src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -2,35 +2,50 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Random;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class Ladder {
 
-  private List<Person> persons;
+  private static final Random RANDOM = new Random();
 
-  private List<Line> lines;
+  private Persons persons;
+  private Lines lines;
 
-  public Ladder() {
+  private Ladder(String[] names, int height, LineStrategy lineStrategy) {
+    this.persons = new Persons(Arrays.stream(names).map(Person::generate).collect(Collectors.toList()));
+    this.lines = new Lines(
```

Comment:
이미 일급컬렉션으로 `Lines`를 만드셨습니다 😄 👍 
`Line`들을 초기화 하는 부분을 `Lines`에게 위임해보시는건 어떨 까요??

`Ladder`에서는 이렇게 준비만 시킨 다면 훨씬 더 코드도 깔끔해지고, 객체들간의 역할도 분명해질 것 같습니다 😄 
```suggestion
    this.lines = Lines.setup(persons.size(), height, lineStrategy);
```

이 부분 개선 검토 부탁 드립니다 🙇